dev/buildtool/halyard_commands.py

```diff
@@ -489,21 +499,34 @@ def _build_release(self, repository):
     # however I dont see how to get at the existing debian metadata and
     # dont want to ommit something
 
+    options = self.options
     git_dir = repository.git_dir
     summary = self.__scm.git.collect_repository_summary(git_dir)
-    args = self.__gradle.get_common_args()
-
-    args.extend(self.__gradle.get_debian_args(
-        'trusty-stable,xenial-stable,bionic-stable'))
-    build_number = self.options.build_number
-    self.__gradle.check_run(
-        args, self, repository, 'publish', 'build-release',
-        version=self.__release_version, build_number=build_number,
-        gradle_dir=git_dir)
-
-    info_path = os.path.join(self.get_output_dir(), 'halyard_info.yml')
-    logging.debug('Writing build information to %s', info_path)
-    write_to_path(summary.to_yaml(), info_path)
+
+
+    config_path = os.path.join(os.path.dirname(os.path.abspath(__file__)),
+                               'cloudbuild', 'debs.yml')
+    substitutions = {'_BRANCH_NAME': options.git_branch,
+                     '_BRANCH_TAG': re.sub(r'\W', '_', options.git_branch),
+                     '_BUILD_NUMBER': options.build_number,
+                     '_IMAGE_NAME': 'halyard',
+                     '_VERSION': summary.version}
+    # Convert it to the format expected by gcloud: "_FOO=bar,_BAZ=qux"
+    substitutions_arg = ','.join('='.join((str(k), str(v))) for k, v in
+                                 substitutions.items())
+    command = ('gcloud builds submit '
+               ' --account={account} --project={project}'
```

Comment:
While in practice this should always have a `--gcb_project` arg, if not provided, this would end up passing `--project=None`, right?

Reply:
No, you actually get an exception thrown if gcb_project isn't set. (How do I know? It's not because I'm a Python master. It is, in fact, just the opposite.)